tests/integration_tests/test_mlflow.py

```diff
@@ -0,0 +1,109 @@
+from mlflow.tracking import MlflowClient
+import py
+
+import optuna
+from optuna.integration.mlflow import MLflowCallback
+
+
+def _objective_func(trial, x_max=1.0):
```

Comment:
To improve readability, we may want to simplify this objective function:
- as @hvy mentions, we can get rid of the `x_max` argument;
- `assert isinstance(z, float)` would not be necessary in this test scenario.

Reply:
ok - removed the assert

Reply:
added it again or I get this error from CI tests:

```
#!/bin/bash -eo pipefail
. venv/bin/activate
mypy .

tests/integration_tests/test_mlflow.py:16: error: Unsupported operand types for + ("float" and "None")
tests/integration_tests/test_mlflow.py:16: error: Unsupported operand types for + ("float" and "str")
tests/integration_tests/test_mlflow.py:16: note: Right operand is of type "Union[None, bool, int, float, str]"
Found 2 errors in 1 file (checked 133 source files)

Exited with code exit status 1
```